programmers/완주하지-못한-선수.js

```diff
@@ -0,0 +1,52 @@
+function solution(part, comp) {
+    
+    let drop = {};
+    let result;
+    
+    for (let i = 0; i < part.length; i++){
+        drop[part[i]] = part.filter(runner => part[i] === runner).length;
+    }
+    
+    for (let j = 0; j < comp.length; j++){
+        drop[comp[j]] -= 1; 
+        if (drop[comp[j]] === 0) delete drop[comp[j]];
+    }
+
+    result = Object.keys(drop)
```

Comment:
```suggestion
    let result = Object.keys(drop);
```
이렇게 마지막에 result를 사용할꺼라면 위에서 선언하지말고 아래에서 선언하는게 더 좋을 것 같습니다ㅎㅎ